test/renderer_test.exs

```diff
@@ -169,4 +169,11 @@ defmodule RendererTest do
     And no spaces</p>
     """, ?\n)
   end
+
+  test "render tag with boolean attribute" do
+    assert render(~s(div [ab="ab" a] a)) == ~s(<div ab="ab" a>a</div>)
+    assert render(~s(div [a b="b"] c)) == ~s(<div a b="b">c</div>)
+    assert render(~S(div ab="#{b} a" a), b: "b") == ~s(<div ab="b a">a</div>)
```

Comment:
There is another test case I've added in comment to PR, much like this one, but with attributes wrapping, it is still failing. =(